torch/csrc/jit/codegen/cuda/lower_loops.cpp

```diff
@@ -303,15 +304,234 @@ void LoopNestGenerator::handle(Expr* expr) {
     popFor();
 }
 
+namespace {
+
+TensorView* findOutputTensor(Expr* expr) {
+  if (expr->outputs().size() != 1) {
+    return nullptr;
+  }
+  auto out = expr->output(0);
+  if (out->getValType() != ValType::TensorView) {
+    return nullptr;
+  }
+  return out->as<TensorView>();
+}
+
+void findTargetTensor(Expr* expr, TensorView*& target, unsigned& score) {
```

Comment:
I'm a bit uncertain what's this function is doing, please comment it.

Reply:
I added some description in the lowering section of the code gen manual. Let me know if that's not clear enough.